### PR TITLE
Rename `_feature_0` to `_whole_cell` and `_feature_1` to `_nuclear`

### LINKED ISSUE #745: Give feature_0 and feature_1 files more informative names
**Is your feature request related to a problem? Please describe.**
The current pipeline produces tifs that are named feature_0 and feature_1, which are not informative

**Describe the solution you'd like**
We should rename these with more informative names, reflecting the fact that they are cell and nuclear segmentation

### PR DESCRIPTION
**Pull request by alex-l-kong**
**What is the purpose of this PR?**

Closes #745. The current suffixes for the segmented images are unclear, they need to be renamed.

**How did you implement your changes**

See title.

### CODE REVIEW

File: ark/utils/example_dataset.py [hunk 1]
```diff
@@ -154,7 +154,7 @@ def get_example_dataset(dataset: str, save_dir: Union[str, pathlib.Path],
 
     example_dataset = ExampleDataset(dataset=dataset, overwrite_existing=overwrite_existing,
                                      cache_dir=None,
-                                     revision="main")
+                                     revision="5db23d178f266fdd9bdc8db637f287eb9eb4224d")
```
Comment (alex-l-kong) [inline ark/utils/example_dataset.py:157]: We'll change this prior to merging into main.

File: ark/utils/example_dataset_test.py [hunk 2]
```diff
@@ -23,7 +23,7 @@ def dataset_download(request) -> Iterator[ExampleDataset]:
     example_dataset: ExampleDataset = ExampleDataset(
         dataset=request.param,
         cache_dir=None,
-        revision="main"
+        revision="5db23d178f266fdd9bdc8db637f287eb9eb4224d"
```
Comment (alex-l-kong) [inline ark/utils/example_dataset_test.py:26]: Same here.

**Review by ngreenwald — CHANGES_REQUESTED** [review]
Changing the example dataset should have automatically uncovered any remaining references in notebooks 1-4, right?. Will you check the spatial notebooks to see if any of them have remaining references to the old format?

**Comment by alex-l-kong** [comment]
> Changing the example dataset should have automatically uncovered any remaining references in notebooks 1-4, right?. Will you check the spatial notebooks to see if any of them have remaining references to the old format?

Yes, and I've verified that all the spatial notebooks are purged of old references.

**Review by ngreenwald — APPROVED** [review]
Looks good

**Comment by alex-l-kong** [comment]
@ngreenwald can you merge this PR for me? It seems like `coveralls` is underreporting.

**Pull request merged**